docs/resources/property_set.md

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Configuration"
+---
+# Artifactory Property Set Resource
+
+Provides an Artifactory Property Set resource. 
+This resource configuration corresponds to 'propertySets' config block in system configuration XML 
+(REST endpoint: artifactory/api/system/configuration).
+
+## Example Usage
+
+```hcl
+resource "artifactory_property_set" "foo" {
+  name 		= "property-set1"
+  visible 	= true
+
+  property {
+    name = "set1property1"
+
+    predefined_value {
+      name 			= "passed-QA"
+      default_value 	= true
+    }
+
+    predefined_value {
+      name 			= "failed-QA"
+      default_value 	= false
+    }
+
+    closed_predefined_values 	= true
+    multiple_choice 			= true
+  }
+
+  property {
+    name = "set1property2"
+
+    predefined_value {
+      name 			= "passed-QA"
+      default_value 	= true
+    }
+
+    predefined_value {
+      name 			= "failed-QA"
+      default_value 	= false
+    }
+
+    closed_predefined_values 	= false
+    multiple_choice 			= false
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Enable SAML SSO. Default value is `true`.
```

Comment:
Incorrect description. Should match what's in the schema.

Reply:
missed that one, fixed